### PR TITLE
Kubernetes: re-enable Windows E2E

### PR DESCRIPTION
**Pull request by jackfrancis**
**What this PR does / why we need it**: Let's add back Windows Kubernetes E2E tests!

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```
Kubernetes: re-enable Windows E2E
```

### CODE REVIEW

File: test/e2e/kubernetes/kubernetes_test.go [hunk 1]
```diff
@@ -635,8 +637,7 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 	})
 
 	Describe("with a windows agent pool", func() {
-		// TODO stabilize this test
-		/*It("should be able to deploy an iis webserver", func() {
+		It("should be able to deploy an iis webserver", func() {
 			if eng.HasWindowsAgents() {
 				r := rand.New(rand.NewSource(time.Now().UnixNano()))
 				deploymentName := fmt.Sprintf("iis-%s-%v", cfg.Name, r.Intn(99999))
```
Comment (PatrickLang) [inline test/e2e/kubernetes/kubernetes_test.go]: line 644 - s/1709-1803/

Comment (PatrickLang) [reply test/e2e/kubernetes/kubernetes_test.go]: we changed the default version to the latest - Windows Server version 1803. Containers are not cross-compatible without enabling Hyper-V isolation which is an alpha feature

**Comment by jackfrancis** [comment]
@PatrickLang so, with the one test re-enabled, we have 3 of 5 clusters succeeding. Or do we know that the test in question won't work as-implemented w/ v1.10 and v1.11?

**Comment by jackfrancis** [comment]
Prefer #3332

**Pull request closed without merging**